docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -42,15 +38,18 @@ You can rename the main process diagram any time if you wish.
 :::note
 A process application must always have a main process.
 That is why the main process diagram cannot be deleted or moved out of the process application.
+
+To replace the main process' content, copy and paste the BPMN XML from the desired diagram.
```

Comment:
This line can (and should!) be deleted. It makes me cringe to read. I'm glad there's a better option

Reply:
Ok, then let me delete it.